### PR TITLE
retrieve study features

### PR DESCRIPTION
**Pull request by anzioka**

### CODE REVIEW

File: src/main/kotlin/com/openlattice/chronicle/controllers/StudyController.kt [hunk 1]
```diff
@@ -428,6 +429,16 @@ class StudyController @Inject constructor(
         return OK()
     }
 
+    @GetMapping(
+        path =[STUDY_ID_PATH + FEATURES_PATH],
+        produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    override fun getStudyFeatures(@PathVariable(STUDY_ID) studyId: UUID): Map<StudyFeature, Map<String, Any>> {
```
Comment (geekbeast) [inline src/main/kotlin/com/openlattice/chronicle/controllers/StudyController.kt:436]: Should we just return Map<String, Any> or keep it consistent with a bulk endpoint? Cc @UnsungHero97

File: src/main/kotlin/com/openlattice/chronicle/services/studies/StudyService.kt [hunk 2]
```diff
@@ -647,6 +653,18 @@ class StudyService(
         }
     }
 
+    override fun getStudyFeatures(studyId: UUID): Map<StudyFeature, Map<String, Any>> {
+        return storageResolver.getPlatformStorage().connection.use { connection ->
```
Comment (geekbeast) [inline src/main/kotlin/com/openlattice/chronicle/services/studies/StudyService.kt:657]: This should be read from the study cache.

**Review by geekbeast — CHANGES_REQUESTED** [review]
Some minor changes, mainly to read from Study Cache when possible.

File: src/main/kotlin/com/openlattice/chronicle/services/studies/StudyService.kt (hunk 2 above)
Comment (anzioka) [reply src/main/kotlin/com/openlattice/chronicle/services/studies/StudyService.kt:657]: Reading from cache seems to return an empty map even though it's not empty
```
chronicle=# select modules from studies;
                                   modules                                    
------------------------------------------------------------------------------
 {"IOS_SENSOR": {}, "CHRONICLE_SURVEYS": {}, "CHRONICLE_DATA_COLLECTION": {}}

```
Screen Shot 2022-04-08 at 10 00 27 AM

**Comment by anzioka** [comment]
Actually this PR is not necessary. Study features are part of the study object response returned to web client

**Pull request closed without merging**